staging/src/k8s.io/kubectl/pkg/cmd/version/skew_warning.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
nit: we're in 2021 :D

Reply:
Whoops, I guess 2020 was so good I forgot to change it :)

Thanks for catching that